src/antares/model/study.py

```
@@ -367,6 +371,35 @@ def wait_job_completion(self, job: Job, time_out: int = 172800) -> None:
         """
         self._run_service.wait_job_completion(job, time_out)
 
+    def read_outputs(self) -> list[Output]:
+        """
+        Load outputs into current study
+
+        Returns: Output list
+        """
+        outputs = self._output_service.read_outputs()
+        self._outputs = {output.name: output for output in outputs}
+        return outputs
+
+    def get_outputs(self) -> dict[str, Output]:
+        """
+        Get outputs of current study
+
+        Returns: (output_id, Output) dictionary
+        """
+        return self._outputs.copy()
+
+    def get_output(self, output_id: str) -> Optional[Output]:
+        """
+        Get a specific output
+
+        Args:
+            output_id: id of the output to get
+
+        Returns: Output with the output_id
+        """
+        return self._outputs.get(output_id)
```

Should be crash instead of returning a None ? I don't know